### PR TITLE
added image component

### PR DESCRIPTION
**Pull request by omjadhav85**
Added img and its variants:
1. Suare and Circular image
2. Different sizes
3. Responsive image

image

### CODE REVIEW

File: styles/components/image.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.img-res {
+  width: 100%;
+  height: 100%;
```
Comment (shuklahimanshu81) [inline styles/components/image.css:3]: style suggested to make responsive : width: 100%; height: auto;

File: styles/components/image.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.img-lg {
+  height: 15.625rem;
+  width: 15.625rem;
+}
+
+.img-round {
+  border-radius: 50%;
+}
```
Comment (shuklahimanshu81) [inline styles/components/image.css]: Suggestion : Create variable of different border radius and use them.

**Review by shuklahimanshu81 — COMMENTED** [review]
Changes suggested. Else looks good to me. Can be merged.

File: styles/components/image.css (hunk 1 above)
Comment (omjadhav85) [reply styles/components/image.css:3]: Thanks for reviewing Himanshu. I tried keeping it as auto but then it overflows its container in my horizontal card. So had to keep it as 100%

File: styles/components/image.css (hunk 2 above)
Comment (omjadhav85) [reply styles/components/image.css]: Thanks! Made changes in the code.

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean. Can be merged

**Pull request merged**